specification/network/resource-manager/readme.md

```diff
@@ -100,6 +100,15 @@ input-file:
   - Microsoft.Network/stable/2024-03-01/vmssNetworkInterface.json
   - Microsoft.Network/stable/2024-03-01/vmssPublicIpAddress.json
   - Microsoft.Network/stable/2024-03-01/webapplicationfirewall.json
+suppressions:
```

Comment:
Could you tell the reason why these suppressions are being added now ? rather than adding at the time of API review ?

Reply:
ARM has already approved the PR's from where LintDiff violations are coming